framework/includes/option-types/multi-picker/class-fw-option-type-multi-picker.php

```diff
@@ -283,6 +288,35 @@ private function prepare_option($id, $option)
 
 		return array_merge($picker_group, $choices_groups);
 	}
+	
+	/**
+	 * Prepare `choices` array.
+	 *
+	 * @since 2.6.11
+	 * @param array $option Options.
+	 * @return array
+	 */
+	protected function prepare_choices($option) {
+		$result = array();
+		$choices = fw_akg('choices', $option);
+
+		if (is_array($choices)) {
+			foreach ($choices as $key => $settings) {
+				if (isset($settings['for']) && isset($settings['options'])) {
+					if (is_array($settings['for'])) {
+						foreach($settings['for'] as $name) {
+							$result[$name] = $settings['options'];
```

Comment:
Probably check here if there's a `$settings[$name]` present. If it does, do a merge instead of a replace.